src/transformers/models/reformer/tokenization_reformer.py

```diff
@@ -167,3 +167,35 @@ def save_vocabulary(self, save_directory: str, filename_prefix: Optional[str] =
             copyfile(self.vocab_file, out_vocab_file)
 
         return (out_vocab_file,)
+
+
+class ReformerByteTokenizer(PreTrainedTokenizer):
```

Comment:
Is that byte tokenizer free of special tokens?

Reply:
The tokenizer is added to the docs, but there isn't really any doc. What is that tokenizer? How can it be used? With what should it be used?

The current full docs looks like this:

![image](https://user-images.githubusercontent.com/30755778/119631222-ae9ed900-bddd-11eb-8f8a-5b0fea0ebf6c.png)

Reply:
If it doesn't need a vocabulary, then it would be nice to indicate this somewhere too - and probably to remove the requirement for the tokenizer to even have a vocabulary file in the first place. This would make saving/loading simpler and wouldn't rely on dummy files.

If it's too hard because it would mean overriding the `from_pretrained` method, then we can keep the workaround documented.